randomizer/Patching/ASMPatcher.py

```diff
@@ -2155,6 +2155,8 @@ def patchAssembly(ROM_COPY, spoiler):
         RemovedBarriersSelected.caves_ice_walls: [266, 267, 265],  # Entrance, Snide, Giant Boulder
         RemovedBarriersSelected.galleon_treasure_room: [0xA2],
         RemovedBarriersSelected.aztec_tiny_temple_ice: [0x45],
+        RemovedBarriersSelected.helm_star_gates: [0x3B],  # Roman Numeral Doors
```

Comment:
This array does permanent flags. Roman numeral doors and punch gates in Helm are temporary flags